nengo_gui/components/netgraph.py

```diff
@@ -469,8 +470,12 @@ def get_extra_info(self, obj):
         elif isinstance(obj, nengo.Ensemble):
             info['dimensions'] = int(obj.size_out)
             info['n_neurons'] = int(obj.n_neurons)
+        elif isinstance(obj, spa.Compare):
+            info['scalar_out'] = True
```

Comment:
Hmm, but on second thought, that more generic approach would cause ```spa.State``` to also have this line, which we don't want.  So let's stick with this special-case for now and worry about it later.   I still think the name of the flag should be changed, though.  :)